src/htcp.cc

```diff
@@ -266,6 +268,22 @@ static void htcpHandleTstRequest(htcpDataHeader *, char *buf, int sz, Ip::Addres
 
 static void htcpHandleTstResponse(htcpDataHeader *, char *, int, Ip::Address &);
 
+static void
+htcpSyncAle(AccessLogEntryPointer &al, const Ip::Address &caddr, int opcode, LogTags logcode, const char *url)
+{
+    if (!al)
+        al = new AccessLogEntry();
+    al->cache.caddr = caddr;
+    al->htcp.opcode = htcpOpcodeStr[opcode];
+    al->cache.code = logcode;
+    al->url = url;
+    // HTCP transactions do not wait
+    al->cache.start_time = current_time;
+    // al->cache.trTime remains zero
+    al->cache.trTime.tv_sec = 0;
```

Comment:
We should either remove the comment (because it duplicates the code) or the two code lines (because they are technically not needed since `trTime` is zero-initialized). Your call.